iroh/src/magicsock/transports/relay/actor.rs

```diff
@@ -680,11 +666,11 @@ impl ActiveRelayActor {
 
     fn handle_relay_msg(&mut self, msg: RelayToClientMsg, state: &mut ConnectedRelayState) {
         match msg {
-            RelayToClientMsg::ReceivedPacket {
-                src_key: remote_node_id,
-                content,
+            RelayToClientMsg::Datagrams {
+                remote_node_id,
+                datagrams,
             } => {
-                trace!(len = %content.len(), "received msg");
+                trace!(len = %datagrams.contents.len(), "received msg");
```

Comment:
Totally a nit, but I don't think you need `%` for this, tracing can record `usize` directly without having to go via `Display`.